pkg/env/env.go

```diff
@@ -442,6 +442,16 @@ func (e *testEnv) execFeature(ctx context.Context, t *testing.T, featName string
 		failed := false
 		for i, assess := range assessments {
 			assessName := assess.Name()
+			if dAssess, ok := assess.(types.DescribableStep); ok {
+				if dAssess.Description() != "" {
+					klog.V(4).InfoS("Processing Assessment", "description", dAssess.Description())
+					defer func() {
+						if t.Failed() {
+							t.Logf("Failed Executing Test: %s", dAssess.Description())
+						}
```

Comment:
```suggestion
					t.Logf("Assessment description: %s", dAssess.Description())
```

Would this work? My goal is that:

* The description is printed in `-test.v` mode even if the test _doesn't_ fail.
* The description is printed (with or without `-test.v`) if the test does fail.

I don't think we need to explicitly state that the test failed - it's fine for the reader to see (roughly):

* TestFoo: Assessment description: ensure that something something...
* TestFoo: FAILED

Reply:
Makes sense. Cleaned it up. What you suggested is much easier.